diff --git a/node_modules/react-native-performance-stats/android/src/main/java/nl/skillnation/perfstats/PerformanceStatsImpl.java b/node_modules/react-native-performance-stats/android/src/main/java/nl/skillnation/perfstats/PerformanceStatsImpl.java
index 621ef85..2491e48 100644
--- a/node_modules/react-native-performance-stats/android/src/main/java/nl/skillnation/perfstats/PerformanceStatsImpl.java
+++ b/node_modules/react-native-performance-stats/android/src/main/java/nl/skillnation/perfstats/PerformanceStatsImpl.java
@@ -91,8 +91,8 @@ public class PerformanceStatsImpl {
             // Collect FPS info
             mTotalFramesDropped += mFrameCallback.getExpectedNumFrames() - mFrameCallback.getNumFrames();
             mTotal4PlusFrameStutters += mFrameCallback.get4PlusFrameStutters();
-            double fps = mFrameCallback.getFPS();
-            double jsFps = mFrameCallback.getJSFPS();
+            double fps = mFrameCallback.getFps();
+            double jsFps = mFrameCallback.getJsFPS();
 
             // Collect system resource usage
             double cpuUsage = 0;
diff --git a/node_modules/react-native-performance-stats/react-native-performance-stats.podspec b/node_modules/react-native-performance-stats/react-native-performance-stats.podspec
index 82dd7fe..232312b 100644
--- a/node_modules/react-native-performance-stats/react-native-performance-stats.podspec
+++ b/node_modules/react-native-performance-stats/react-native-performance-stats.podspec
@@ -2,7 +2,6 @@ require "json"
 
 package = JSON.parse(File.read(File.join(__dir__, "package.json")))
 
-folly_version = '2021.06.28.00-v2'
 folly_compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32'
 
 Pod::Spec.new do |s|
@@ -29,7 +28,7 @@ Pod::Spec.new do |s|
         }
 
         s.dependency "React-Codegen"
-        s.dependency "RCT-Folly", folly_version
+        s.dependency "RCT-Folly"
         s.dependency "RCTRequired"
         s.dependency "RCTTypeSafety"
         s.dependency "ReactCommon/turbomodule/core"
